data/coalition/coalition missions.txt

```diff
@@ -2613,7 +2613,7 @@ mission "Coalition: Expeditions Past 6"
 			`Sedlitaris starts triangulating to find the next pylon again, and brings you to a tall mountain where some of the local dragons appear to have made some form of nest or breeding ground. You do your best to avoid them as you fly around the mountain, and Sedlitaris continues to send signals to find the pylon.`
 			`	"Inside the mountain, it could be. From there, the countersignals are coming." She stops for a few minutes and looks at the view of the mountain on your monitors. She tries again, and asks that you get closer. When you're finishing yet another lap, she asks that you stop your ship, keeping it steady in the air. "Zoom your cameras here, could you, Captain?" she asks, pointing to the middle of a monitor. You zoom in and see a circle of large rocks, surrounded by dragon nests. Atop the rocks are some patches of foliage, and in the middle lies a small, dark, tower-like structure, stuck in the rocks by one end. After looking at it for a few seconds, you both realize it's an upside-down pylon. "Taken it as some form of token, the dragons must have," Sedlitaris says. "Hopes of retrieving some of the pylons physically, I had, but prefer to dig out the glacier for the previous one than test my luck against the dragons, I do. Unlock this one's data remotely, I will."`
 			`	A dragon perches on your ship as she begins the transfer. It tries to bite down on the engines, but you start moving your ship again to shake it off. You avoid other dragons as they fly by your ship until the download is complete, and head straight for the spaceport city.`
-			`	Once your ship has safely landed on the hangars, Sedlitaris plays the audio log: "Log started. Scoured the planet with our searches, we have, but no trace of the previously visited ruins, or any other sign of the group that lived here, we found. Assured me that all our ships' navigation systems are working properly, the engineers have, but at the coordinates where they should be, the ruins are not, as if they had never been there. A confusing disappointment, this was, but continue with the mission, we must. Heading far north, to Summit, we are. The last point before regrouping with the split group, it will be."`
+			`	Once your ship has safely landed at the hangars, Sedlitaris plays the audio log: "Log started. Scoured the planet with our searches, we have, but no trace of the previously visited ruins, or any other sign of the group that lived here, we found. Assured me that all our ships' navigation systems are working properly, the engineers have, but at the coordinates where they should be, the ruins are not, as if they had never been there. A confusing disappointment, this was, but continue with the mission, we must. Heading far north, to Summit, we are. The last point before regrouping with the split group, it will be."`
```

Comment:
```suggestion
			`	Once your ship has safely landed, Sedlitaris plays the audio log: "Log started. Scoured the planet with our searches, we have, but no trace of the previously visited ruins, or any other sign of the group that lived here, we found. Assured me that all our ships' navigation systems are working properly, the engineers have, but at the coordinates where they should be, the ruins are not, as if they had never been there. A confusing disappointment, this was, but continue with the mission, we must. Heading far north, to Summit, we are. The last point before regrouping with the split group, it will be."`
```
I think this sentence flows better with the mention of hangars simply removed. The previous line mentions that you're heading straight for the spaceport so it seems unnecessary, and a little clunky, to specify where you're landing.

Reply:
I'm marginally not in favor of this change. Yes, it makes it more concise; but I also like having detail about what's going on, and knowing that I'm landing at the hangars instead of, say, on a concrete pad in a field or in the outfitters, does make a difference to what I feel and imagine. Painting an appropriate picture is more important that being concise.

It's not a strong opinion, if the writing team deems it something that should be done, OK. But I don't think it is necessary.